doc/ref/configuration/master.rst

```diff
@@ -408,6 +408,19 @@ to False.
 
     color: False
 
+.. conf_master:: color_theme
+
+``color_theme``
+---------
+
+Default: ``{}``
```

Comment:
I don't think these defaults are correct. I think this should be an empty string since setting the color_theme is a file path in the example below.

Reply:
so true... not sure why I used this ;)